src/consumer/runner.js

```diff
@@ -167,7 +167,7 @@ module.exports = class Runner extends EventEmitter {
 
       this.consumerGroup.resolveOffset({ topic, partition, offset: message.offset })
       await this.consumerGroup.heartbeat({ interval: this.heartbeatInterval })
-      await this.consumerGroup.commitOffsetsIfNecessary()
+      await this.autoCommitOffsetsIfNecessary()
```

Comment:
I swear I've seen an issue about this, but I couldn't find it. At the end of the fetch loop, we were committing if either of the thresholds had been exceeded, even if `autoCommit` was false.

Reply:
https://github.com/tulios/kafkajs/pull/783#discussion_r450676604 ?

Reply:
Yes! That was it.